app/models/generic_work.rb

```diff
@@ -2,12 +2,30 @@
 
 class GenericWork < ActiveFedora::Base
   include ::Hyrax::WorkBehavior
-  include ::Hyrax::BasicMetadata
   include IiifPrint.model_configuration(
     pdf_split_child_model: self
   )
 
-  validates :title, presence: { message: 'Your work must have a title.' }
+  # rubocop:disable Metrics/LineLength
+  property :institution, predicate: ::RDF::URI.new("http://test.hyku.test/generic_work#institution"), multiple: false do |index|
+    index.as :stored_searchable, :facetable
+  end
+  # rubocop:enable Metrics/LineLength
```

Comment:
You can remove the rubocop disable if you format like this:
```suggestion
property :institution,
          predicate: ::RDF::URI.intern('https://atla.com/terms/Institution'),
          multiple: false do |index|
            index.as :stored_searchable
          end
```